### PR TITLE
vlad-fix-part4

### PR DESCRIPTION
**Pull request by vlad666spot**
Fixed the next API calls:
- AWS EG Stateful
- AWS EG Beanstalk
- AWS ECS

### CODE REVIEW

File: api/services/elastigroup/aws/paths/ecsBeanstalkImport.yaml [hunk 1]
```diff
@@ -10,21 +10,28 @@ get:
   - "Elastigroup AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: path
```
Comment (sagishoshan) [inline api/services/elastigroup/aws/paths/ecsBeanstalkImport.yaml:13]: @vlad666spot I checked http://help.dev.spotinst.com/spotinst-api/elastigroup/amazon-web-services/beanstalk-api/get-beanstalk-configuration/
it seems that **groupId** is not relevant.
so it's good that you removed the ref, but why did you add it explicitly afterward? let's remove it

File: api/services/elastigroup/aws/paths/ecsGroupClusterRollId.yaml [hunk 2]
```diff
@@ -7,7 +7,14 @@ post:
   - "Elastigroup AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: path
```
Comment (sagishoshan) [inline api/services/elastigroup/aws/paths/ecsGroupClusterRollId.yaml:10]: @vlad666spot why not using the reference?

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/elastigroup/aws/paths/ecsBeanstalkImport.yaml (hunk 1 above)
Comment (vlad666spot) [reply api/services/elastigroup/aws/paths/ecsBeanstalkImport.yaml]: fixed

File: api/services/elastigroup/aws/paths/ecsGroupClusterRollId.yaml (hunk 2 above)
Comment (vlad666spot) [reply api/services/elastigroup/aws/paths/ecsGroupClusterRollId.yaml]: fixed

**Review by sagishoshan — APPROVED** [review]
